Vagrantfile

```diff
@@ -0,0 +1,39 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+    config.vm.box = "manager-for-lustre/centos74-1708-base"
+    config.vm.synced_folder ".", "/vagrant", type: "virtualbox"
+    config.vm.provider "virtualbox" do |vb|
+      vb.customize ["modifyvm", :id, "--memory", "512"]
+      vb.name = "libzfs"
+      
+  
+      file_to_disk = './tmp/medium_disk.vdi'
+  
+      unless File.exist?(file_to_disk)
+        vb.customize ['createhd', '--filename', file_to_disk, '--size', 500 * 1024]
+      end
+
+      vb.customize ['storageattach', :id, '--storagectl', 'SATA Controller', '--port', 1, '--device', 0, '--type', 'hdd', '--medium', file_to_disk]
+      vb.customize ['setextradata', :id, 'VBoxInternal/Devices/ahci/0/Config/Port1/SerialNumber', '081118FC1221NCJ6G8GG']
+    end
+  
+    config.vm.boot_timeout = 600
+  
+    config.vm.provision "shell", inline: <<-SHELL
+        yum -y install yum-plugin-copr epel-release
+        yum-config-manager --add-repo https://downloads.hpdd.intel.com/public/lustre/lustre-2.10.1/el7/server
```

Comment:
Should we be using ZoL for our source of ZFS?

I think it would be better to be using canonical builds rather than unofficial re-builds and using ZoL future-proofs us against the Lustre team ceasing building of ZFS if/when they refer to upstream also.

Reply:
I thought about this initially, but decided against it for two reasons:

1. We are not using the ZOL repo at the moment.
2. The ZOL repo does not hold old versions.

Reply:
Just to add to this, once Lustre switches to the ZOL repo proper, I agree we should switch to ZOL here as well.